Thrive.csproj

```diff
@@ -634,7 +642,11 @@
     <Compile Include="src\microbe_stage\PatchNameOverlay.cs" />
     <Compile Include="src\microbe_stage\PhagocytosisPhase.cs" />
     <Compile Include="src\microbe_stage\PlacedOrganelle.cs" />
+    <Compile Include="src\statistics_tracking\ReproductionStatistic.cs" />
```

Comment:
I think this doesn't really deserve a top level folder. I'd put this folder in the general folder so there would be `src/general/statistics_tracking` as the path for these files.